### ISSUE TITLE
Visual Accounts

### ISSUE THREAD

**Issue by VoltraInc** [microsoft/vscode#186932]
# Visual Accounts
#### Features:
1. Ability to sync extensions, files and themes on different devices
2. Ability to share files between users by friending each other

#### How will it work?
When Visual Studio Code is opened, the user should choose whether to login or use Visual Studio Code as a guest with all the normal features. If a user chooses to continue as user. They are prompted whether to login or signup. Signing up will prompt the user with a display name, username, email and password. Logging in will prompt the user with email/username and password. After logging in, the user will choose if they want to sync extensions, then themes, then last opened folder with all the files. A user can be friended or friend others, can be followed or follow others and can block and be blocked by others. The user settings will contain the ability to add/change profile picture, email, username, display name or password.

If you have any confusions and would like to know more, please email me at [selimwalyy@gmail.com](mailto:selimwalyy@gmail.com).

**Comment by IllusionMH** [comment]
VS Code already has [Setting Sync](https://code.visualstudio.com/docs/editor/settings-sync) and [Profiles](https://code.visualstudio.com/docs/editor/profiles)

Other features:
* multiuser instance - each OS user should have its own settings as they are stored in user folder otherwise portable version should be used.

* guests, friends etc. looks way out of scope of code editor.
